src/views/Farms/components/FarmTable/Actions/HarvestAction.tsx

```diff
@@ -49,7 +49,7 @@ const HarvestAction: React.FunctionComponent<HarvestActionProps> = ({ pid, userD
     <ActionContainer>
       <ActionTitles>
         <Title>CAKE </Title>
-        <Subtle>{t('EARNED')}</Subtle>
+        <Subtle>{t('Earned').toUpperCase()}</Subtle>
```

Comment:
The `<Subtle />` component is not necessary. Don't worry about changing it now so for this case we can keep `toUpperCase`. However, can you add a `TODO:` for some dev to remove this component in favor of `<Text />`

Reply:
I added TODO